scribus/plugins/import/pdf/slaoutput.h

```diff
@@ -306,6 +390,13 @@ class SlaOutputDev : public OutputDev
 
 	void createImageFrame(QImage& image, GfxState *state, int numColorComponents);
 
+	//PDF Textbox framework	

```

Comment:
if it's a framework, can it be enclosed in a class?

Reply:
I've created a new class and migrated any framework compnents.